optimize/components/userguide/additional-features/event-based-processes.md

```diff
@@ -1,248 +0,0 @@
----
-id: event-based-processes
-title: Event-based processes
-description: Create and analyze reports backed by ingested events.
----
-
-<span class="badge badge--platform">Camunda 7 only</span>
-
-## Overview
```

Comment:
There is another page here: https://preview.docs.camunda.cloud/pr-4785/optimize/next/self-managed/optimize-deployment/advanced-features/engine-data-deletion/

I think that this is pretty C7 exclusive too (although we never marked it as so). With this in mind, I propose we completely delete the whole "additional features" section from the docs. Wdyt?